pkg/agent/core/state.go

```diff
@@ -130,9 +129,7 @@ type state struct {
 	// NeonVM records all state relevant to the NeonVM k8s API
 	NeonVM neonvmState
 
-	Metrics *SystemMetrics
-
-	LFCMetrics *LFCMetrics
+	Metrics A
```

Comment:
Shouldn't we rename it here as well?

Reply:
Yup, good catch, thanks. Did that rename in 8d053b1cc3c57202ed5e1fcceb1e2ed26d67dc2f